### PR TITLE
Disable property and view in analytics when not setup.

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #1039.

## Relevant technical choices

Disables property and view if nothing is selected for them to use.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by felixarntz** [comment]
@tofumatt @aaemnnosttv Actually, in this screenshot both the property and the profile dropdown _should_ be disabled.

In order to select a property, you must select an account first.
In order to select a profile, you must select a property first.

**Comment by aaemnnosttv** [comment]
That makes sense @felixarntz. In that case, @tofumatt will need to approve and include the visual changes in addition to making the necessary change for the profile dropdown.

File: assets/js/modules/analytics/setup.js [hunk 1]
```diff
@@ -796,12 +796,8 @@ class AnalyticsSetup extends Component {
...
 			isEditing,
 		} = this.props;
-		const disabledProfile = ! isEditing;
-
-		let disabledProperty = ! isEditing;
-		if ( existingTag && selectedProperty ) {
-			disabledProperty = true;
-		}
+		const disabledProperty = ! isEditing || parseInt( selectedAccount, 10 ) < 1 || ! selectedAccount || ( existingTag && selectedProperty );
+		const disabledProfile = ! isEditing || ! selectedProperty;
```
Comment (aaemnnosttv) [inline assets/js/modules/analytics/setup.js]:
```suggestion
		const disabledProfile = ! isEditing || ~~selectedProperty < 1;
```

Comment (felixarntz) [reply assets/js/modules/analytics/setup.js]: @aaemnnosttv How about `disabledProfile = ! isEditing || ! selectedProperty || disabledProperty;`?

Comment (tofumatt) [reply assets/js/modules/analytics/setup.js]: We want to test for `0` or `-1` as well; they're magic constant values used to set up a new property. If that's enabled we shouldn't allow the user to select a view.

**Comment by tofumatt** [comment]
All set for another review; VRTs should be passing now.

**Comment by tofumatt** [comment]
Looking into these E2E test failures now…

**Comment by tofumatt** [comment]
All set now! 😄

**Comment by tofumatt** [comment]
Okay, _now_ this is good to review! Thanks for your patience on this one! 😅

**Comment by aaemnnosttv** [comment]
@googlebot I consent.

**Pull request merged**